### PR TITLE
[546] Redesign of AffiliateManagerCard element

### PR DESCRIPTION
**Pull request by rlaco-qualityunit**
**Changes proposed in this Pull Request**
Visual changes in AffiliateManagerCard element (smaller image, darken subtitle text, "show profile" button)
New php function for inline svg images "get_svg_image"
New scss mixin for hover

**Testing instructions**
Check design changes and responsive

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] My commits follow the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Close QualityUnit/web-issues#546

### CODE REVIEW

File: templates/content-single-ms_directory.php [hunk 1]
```diff
@@ -670,10 +670,9 @@ function manager_industry( $manager_industry ) {
 										<?php
 										if ( isset( $url ) ) {
 											?>
-											<a href="<?= esc_url( add_query_arg( array( 'directory_name' => get_the_title(), 'directory_url' => get_the_permalink() ), $url ) ); // @codingStandardsIgnoreLine ?>" class="AffiliateManagerCard__image--showProfile" title="<?= esc_html( $url_title ); ?>">
-									<?= $picture; // @codingStandardsIgnoreLine ?>
-									<span><?php _e( 'Show profile', 'ms' ); ?></span>
-								</a>
+                                            <a href="<?= esc_url( add_query_arg( array( 'directory_name' => get_the_title(), 'directory_url' => get_the_permalink() ), $url ) ); // @codingStandardsIgnoreLine ?>" class="AffiliateManagerCard__image" title="<?= esc_html( $url_title ); ?>">
```
Comment (jremen) [inline templates/content-single-ms_directory.php:673]: Please fix intendation of <a href part (too many spaces, use less intendation using tabs)

File: templates/content-single-ms_directory.php [hunk 2]
```diff
@@ -724,6 +723,16 @@ function manager_industry( $manager_industry ) {
 											?>
 										</ul>
 									</div>
+
+									<?php
+									if ( isset( $url ) ) {
+										?>
+										<div class="AffiliateManagerCard__showProfile">
+                                            <a href="<?= esc_url( add_query_arg( array('directory_name' => get_the_title(), 'directory_url' => get_the_permalink() ), $url ) ); ?>" class="learn-more" title="<?= esc_html( $url_title ); ?>"><?php _e( 'Show profile', 'ms'); ?><?php echo QualityUnit\Extras\get_svg_image( 'arrow_small_right' ); // @codingStandardsIgnoreLine?></a>
```
Comment (jremen) [inline templates/content-single-ms_directory.php:731]: Please fix intendation of <a href part (too many spaces, use less intendation using tabs)

**Review by jremen — COMMENTED** [review]
Please fix intendation of <a href part (too many spaces, use less intendation using tabs)

File: templates/content-single-ms_directory.php (hunk 2 above)
Comment (rlaco-qualityunit) [reply templates/content-single-ms_directory.php]: fixed

File: templates/content-single-ms_directory.php (hunk 1 above)
Comment (rlaco-qualityunit) [reply templates/content-single-ms_directory.php:673]: fixed

**Comment by rlaco-qualityunit** [comment]
> Please fix intendation of <a href part (too many spaces, use less intendation using tabs)

@jremen all idents was fixed

**Review by mvstepanek — APPROVED** [review]

**Pull request merged**